### ISSUE TITLE
 [New feature] Add Slider.kz (music downloader)

### ISSUE THREAD

**Issue by TtuxX** [shiningw/ncdownloader#32]
Hi @shiningw and thank you for the work you put in this great Downloader ! :)

I know it might be out of scope - and would understand it - but would it be possible scape a music download website too ?

You could maybe rename the "Search Torrents" button to make it become "Search Torrents/Musics" and add [Slider.kz](https://slider.kz) as a 3rd option in the dropdown (after TPB and Bitsearch) ?

I coded a bash music downloader based on this website:
1. It scrapes the musics results Url lists using this link: http://slider.kz/vk_auth.php?q= 
2. It extracts the music name and url, BUT one important data is missing: the file size. The file size gives a good idea of the music length/weight on hard drive/quality.
3. Thus, my script quickly fetch the Content-Length of each url in parallel
4. Finally results are displayed with their size, sorted from the largest file to the smallest one.
5. When the file is downloaded, the original displayed file name is kept, instead of the download name that includes the slider.kz "watermark" text.

Do you think you could implement this ?:)

 It would really be awesome as I download musics on a daily basis but have to SSH to use my tool.
 My tool then moves the downloaded file to Nextcloud and modify its permissions to www and scans it using OCC to ensure it is properly added to the filecache DB.

I can of course share with you the full source code of my music downloader!

Kind regards

**Comment by shiningw** [comment]
Hi,TtuxX
 It sounds interesting! Please do send me a copy of  the bash script and I will take a look at it

**Comment by TtuxX** [comment]
May I have your mail address ? :)

**Comment by shiningw** [comment]
Hi,
I've taken a look at the site, and it shouldn't be difficult to implement. No worry, I can figure it out myself. Thanks for your help!
By the way, Do you know of any other good free music sites?

**Comment by TtuxX** [comment]
VKontakte is the underlying database under [Slider.kz](https://slider.kz/) 
Music downloading websites often are sourcing their songs in this same database, which is - to my knowledge - the largest direct download database available online :)

Please find hereunder some other websites:

_VKontakte_:
https://patefon.co
http://mp3-vk.ru

_VKontakte + Youtube_:
https://beemp3s.net (results appear to be mixed with VK results top and Youtube-converted musics after)
https://mp3juice.info (results appear to be mixed with VK results top and Youtube-converted musics after)

Youtube, Dailymotion, Vimeo, Facebook, Instagram, Metacafe, VK, SoundCloud, AOL, Vevo, Google Drive and Twitter:
https://www.y2mp3.net/

**Comment by TtuxX** [comment]
> support added

Thank you very much ! :) Is it possible to replace the underscores by spaces characters for downloaded files ?

**Comment by shiningw** [comment]
> > support added
> 
> Thank you very much ! :) Is it possible to replace the underscores by spaces characters for downloaded files ?

I will consider fixing it in next version
